fem/datacollection.cpp

```diff
@@ -522,9 +624,10 @@ std::string VisItDataCollection::GetVisItRootString()
    mtags["spatial_dim"] = picojson::value(to_string(spatial_dim));
    mtags["topo_dim"] = picojson::value(to_string(topo_dim));
    mtags["max_lods"] = picojson::value(to_string(visit_max_levels_of_detail));
-   mesh["path"] = picojson::value(path_str + ((format==0)?"":"p") + "mesh" +
-                                  file_ext_format);
+   mesh["path"] = picojson::value(path_str + ((format==SERIAL_FORMAT)?"":"p") +
```

Comment:
It looks like there are several places in ``[Visit]DataCollection`` (at least three) that deal with appending "mesh" or "pmesh" to the mesh path, and this is handled in at least two different ways.

A utility function that returns the correct string would help ensure consistency.

Reply:
Fixed.